apps/trading/components/liquidity-supplied/liquidity-supplied.tsx

```diff
@@ -131,8 +132,16 @@ export const MarketLiquiditySupplied = ({
       description={description}
       testId="liquidity-supplied"
     >
-      <div>
-        ({formatNumberPercentage(percentage, 2)}) {supplied}
+      <div className="flex flex-inline gap-1">
+        <span
+          className={classNames('rounded-full h-2 w-2 mt-1 mr-1', {
```

Comment:
chore (blocking): use Indicator component from the toolkit

Reply:
forgot we have it. updated